crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -748,6 +748,20 @@ en:
       non_json_file: 'Unprocessable chef file found'
       invalid_file_extension: 'Filename needs to be a valid tarball with a .tar.gz file extension'
 
+  sanities:
+    show:
+      header: 'Sanity Check Failures'
+      notice: |
+        There are errors that must be resolved. Please restart Crowbar after solving the errors.
```

Comment:
Do we actually really need to restart crowbar? Or is it okay to just do the sanity checks again? (there's a button in the ui for that, if I understand correctly)